examples/kube/centralized-logging/efk/cleanup.sh

```diff
@@ -16,13 +16,13 @@ source ${CCPROOT}/examples/common.sh
 echo_info "Cleaning up.."
 
 ${CCP_CLI?} delete --namespace=kube-system \
-    service,statefulset,deployment,pod,daemonset,configmap,secret,serviceaccount,pvc \
+    service,deployment,pod,daemonset,configmap,secret,serviceaccount,pvc \
     --selector=k8s-app=elasticsearch-logging
```

Comment:
The StatefulSets in this directory seem to be for Elasticsearch, #316. They should probably stay unless we're removing this entire example as well.

Reply:
Let's leave those in for now.

Reply:
sounds good, i'll put the statefulsets back in for Elasticsearch